docs/celeborn_cli.md

```diff
@@ -118,6 +121,9 @@ Usage: celeborn-cli master [-hV] [--apps=appId] [--auth-header=authHeader]
       --config-tenant=tenant_id
                              The tenant id of TENANT or TENANT_USER level.
       --delete-apps          Delete resource of an application.
+      --delete-configs=c1,c2,c3...
```

Comment:
this option looks confused with `--delete-dynamic-conf`.

How about `--dynamic-configs`.


```
--delete-dynamic-conf --dynamic-configs=c1,c2,c3
--upsert-dynamic-conf --dynamic-configs=c1,c2,c3
```